api/src/opentrons/protocol_api/instrument_context.py

```diff
@@ -1624,3 +1624,47 @@ def configure_for_volume(self, volume: float) -> None:
         if last_location and isinstance(last_location.labware, labware.Well):
             self.move_to(last_location.labware.top())
         self._core.configure_for_volume(volume)
+
+    @requires_version(2, 16)
+    def prepare_to_aspirate(self) -> None:
+        """Prepare a pipette for aspiration.
+
+        Before a pipette can aspirate into an empty tip, the plunger must be in its
+        bottom position. After dropping a tip or blowing out, the plunger will be in a
+        different position. This function moves the plunger to the bottom position,
+        regardless of its current position, to make sure that the pipette is ready to
+        aspirate.
+
+        You rarely need to call this function. The API automatically prepares the
+        pipette for aspiration as part of other commands:
+
+            - After picking up a tip with :py:meth:`.pick_up_tip`.
+            - When calling :py:meth:`.aspirate`, if the pipette isn't already prepared.
+              If the pipette is in a well, it will move out of the well, move the plunger,
+              and then move back.
+
+        Use ``prepare_to_aspirate`` when you need to control exactly when the plunger
+        motion will happen. A common use case is a pre-wetting routine, which requires
+        preparing for aspiration, moving into a well, and then aspirating *without
+        leaving the well*::
+
+             pipette.move_to(well.top(z=-5))
+             pipette.delay(5)
+             pipette.mix(10, 10)
+             pipette.move_to(well.top(z=5))
+             pipette.blow_out()
+             pipette.prepare_to_aspirate()
+             pipette.move_to(well.top(z=-5))
+             pipette.delay(5)
+             pipette.aspirate(10, well.top(z=-5))
```

Comment:
The negative sign in `well.top(z=-5)` vs `well.top(z=5)` seems easy to miss. Maybe let's use `well.bottom` for the submerged parts, to draw a clearer distinction.

```suggestion
             pipette.move_to(well.bottom(z=2))
             pipette.delay(5)
             pipette.mix(10, 10)
             pipette.move_to(well.top(z=5))
             pipette.blow_out()
             pipette.prepare_to_aspirate()
             pipette.move_to(well.bottom(z=2))
             pipette.delay(5)
             pipette.aspirate(10, well.bottom(z=2))
```